tests/conftest.py

```diff
@@ -338,3 +338,31 @@ def pytest_exception_interact(call: pytest.CallInfo[Any]):
     @pytest.hookimpl(tryfirst=True)
     def pytest_internalerror(excinfo: pytest.ExceptionInfo[Any]):
         raise excinfo.value
+
+
+@pytest.fixture
+def mock_numtracker_server() -> responses.RequestsMock:  # type: ignore
```

Comment:
Could: Rather than `type: ignore`, be honest about the function is returning

```suggestion
def mock_numtracker_server() -> Iterable[responses.RequestsMock]
```

Can then also remove the ignore from the `return` statement

Reply:
Done.